src/logic/calculate.js

```diff
@@ -1,32 +1,69 @@
+// import { string, number } from 'prop-types';
 import operate from './operate';
 
 const calculate = (buttonName, data) => {
   let { total, next, operation } = data;
-  const symbolOperations = ['AC', 'x', '+', '-', '÷', '%'];
+  const symbolOperations = ['x', '+', '-', '÷', '%', '='];
+  const numbers = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
 
   switch (buttonName) {
     case 'AC':
-      total = '0';
-      next = '0';
-      operation = '';
+      total = null;
+      next = null;
+      operation = null;
       break;
     case '+/-':
       total *= -1;
       total *= -1;
```

Comment:
Change the second **total** to **next**, so the operation can work properly. Currently, the button isn't working.